pkg/crd/kubermatic/v1/settings.go

```diff
@@ -56,6 +56,8 @@ type SettingSpec struct {
 	RestrictProjectCreation     bool           `json:"restrictProjectCreation"`
 	EnableExternalClusterImport bool           `json:"enableExternalClusterImport"`
 
+	MachineDeploymentVMResourceQuota MachineDeploymentVMResourceQuota `json:"clusterVMResourceQuota"`
```

Comment:
json tag is different than the field name, is that done on purpose?

Reply:
mistake, thanks

Reply:
done